### ISSUE TITLE
Use Pydantic v2

### ISSUE THREAD

**Issue by aersam** [ilyachch/md_docs-trans-app#78]
pydantic v2 is out for some time, it's great, use it :)

**Comment by ilyachch** [comment]
1. It is not a bug
2. It will not bring any new functions
3. Pydantic 1 is still supported

I think it is "nice to have", but not "must have"

> Do not fix what is not broken

**Comment by aersam** [comment]
> 1. It is not a bug
> 
>     2. It will not bring any new functions
> 
>     3. Pydantic 1 is still supported
> 
> 
> I think it is "nice to have", but not "must have"
> 
> > Do not fix what is not broken

True, I know what you mean. However it's an issue if you have a project that does depend on v2 and you want to add md_markdown as dependency

**Comment by ilyachch** [comment]
> > 1. It is not a bug
> >    
> >    2. It will not bring any new functions
> >    3. Pydantic 1 is still supported
> > 
> > I think it is "nice to have", but not "must have"
> > > Do not fix what is not broken
> 
> True, I know what you mean. However it's an issue if you have a project that does depend on v2 and you want to add md_markdown as dependency

Could you give an example of md_translate usage as dependency, please? In my mind it is just cli utility

If there are any usecases, I think, I could adapt it better for inclusion as library

**Comment by aersam** [comment]
I use it as cli, but install it using poetry as dev dependency so that all dev's have it installed in the venv right away.

**Comment by ilyachch** [comment]
> I use it as cli, but install it using poetry as dev dependency so that all dev's have it installed in the venv right away.

In this case I think, it is better to make pydantic version as extras:

```
pip install md-translate  # For first version of pydantic
OR
pip install md-translate[pydantic2]  # For second version of pydantic
```

or conversely

```
pip install md-translate[pydantic1]  # For first version of pydantic
OR
pip install md-translate  # For second version of pydantic
```
